_posts/2015-02-23-integrating-with-poynt-cloud-apis.md

```diff
@@ -22,25 +22,47 @@ To consume Poynt APIs you must:
+
+9. Complete the application information by clicking on "Edit app details" and filling the required fields. Click on "Save" button.<br/>
+<img class="shadow"  src="{{site.url}}/developer/assets/poynt_net_create_new_app_info.png" alt="Fill Application Info" height="500"/>
+
+10. Click on "Cloud Permissions", enter the OAuth callback URL and edit the permissions. Click on "Save" button.<br/>
+<img class="shadow"  src="{{site.url}}/developer/assets/poynt_net_create_new_app_info.png" alt="Fill Application Info" height="500"/>
+
+11. Obtain merchant's permission to access Poynt APIs on behalf of them (access their data)
     1. Construct Poynt Authorization url to redirect the merchant to:
         - e.g. [https://poynt.net/applications/authorize?redirect_uri=http%3A%2F%2Fgetdemoapp.com%2Freturn_from_poynt&client_id=urn%3Aaid%3A5b4e9993-c815-4070-9859-35f48e492b4f&context=python-test-script](https://poynt.net/applications/authorize?redirect_uri=http%3A%2F%2Fgetdemoapp.com%2Freturn_from_poynt&client_id=urn%3Aaid%3A5b4e9993-c815-4070-9859-35f48e492b4f&context=python-test-script)
         - where `redirect_uri` is an HTTPS page hosted on your own site that will be called with the OAuth authorization code. (Note that this should match the url you've configured in your application settings)
         - `client_id` is your newly created application id starting with urn:aid:
         - `context` is any string value that you would like passed to your callback URL, e.g. a session ID or a user ID on your side so you can match it with the authenticated merchant. This is optional.
-    2. Redirect the merchant to Poynt Authorization url to obtain the necessary permission. The merchant will be prompted to allow access to the resources you specified for your app. If the merchant has already given you permission, the page will automatically redirect to the next step.
-        - Note for development you can use your own test merchant account to give permission and experiment with the APIs.
-        ![Authorization Page]({{site.url}}/developer/assets/authorize_screenshot.png){:height="390" width="600"}
-    3. Merchant gets redirected back to your redirect_uri with `code`, `status`, `context` (optional) and merchant's `businessId`.
+    2. Redirect the merchant to Poynt Authorization url to obtain the necessary permission. The merchant needs to login to authorize.<br/>
+      Note for development you can use your own test merchant account to give permission and experiment with the APIs. <br/> 
+      <img class="shadow"  src="{{site.url}}/developer/assets/poynt_net_login_developer.png" alt="Merchant Sign In" height="500"/><br/>
+      
+    3.  The merchant will be prompted to allow access to the resources you specified for your app.  If the merchant has already given you permission, the page will automatically redirect to the next step. <br/>
+        
+          1. Select the test merchant<br/>
```

Comment:
Should add a note that if the developer logs in with their credentials they will see this screen and will need to select the test merchant to authorize